docs/modules/clients/pages/java.adoc

```diff
@@ -1715,205 +1447,140 @@ YAML::
+identify themselves (to other members) and Hazelcast clients use trustStores to
+define which members they can trust. The clients also have their own keyStores, and
+members have their own trustStores, so that members can
+know which clients they can trust. For more information, see xref:security:tls-ssl.adoc#mutual-authentication[Mutual authentication].
 
-[tabs] 
-==== 
-XML:: 
-+ 
--- 
-[source,xml]
-----
-<hazelcast-client-failover>
-    <try-count>4</try-count>
-    <clients>
-        <client>client-config1.xml</client>
-        <client>client-config2.xml</client>
-        <client>client-config3.xml</client>
-    </clients>
-</hazelcast-client-failover>
-----
---
-
-YAML::
-+
-[source,yaml]
-----
-hazelcast-client-failover:
-  try-count: 4
-  clients:
-    - client-config1.yaml
-    - client-config2.yaml
-    - client-config3.yaml
-----
-====
+// check standalone - is config different or is the content pulled from above linked page?
 
-And let's say the client is disconnected from the cluster
-whose configuration is given by `client-config2.xml`.
-Then, the client tries to connect to the next cluster in this list,
-whose configuration is given by `client-config3.xml`. When the end of the list is reached,
-which is so in this example, and the client could not connect to `client-config3`,
-then `try-count` is incremented and the client continues to try to connect starting with `client-config1`.
+//ClientConfig clientConfig = new ClientConfig();
+//clientConfig.getNetworkConfig().setSSLConfig(sslConfig);
+//```
+//Please note that the paths in the properties here are *absolute paths* to the resources in classpath.
 
-This iteration continues until the client connects to a cluster or `try-count` is reached to the configured value.
-When the iteration reaches this value and the client still could not connect to a cluster,
-it shuts down. Note that, if `try-count` was set to `1` in the above example,
-and the client could not connect to `client-config3`, it would shut down since
-it already tried once to connect to an alternative cluster.
+//To enable mutual authentication on the client, add to the properties:
+//```java
+//properties.setProperty("keyFile", "/path/client.pem");
+//properties.setProperty("keyCertChainFile", "/path/client.crt");
+//```
+//To use the OpenSSL engine instead of the Basic SSL context,
+//replace the SSL context factory class name and implementation as follows:
+//```java
+//sslConfig.setFactoryClassName(OpenSSLEngineFactory.class.getName())
+//        .setFactoryImplementation(new OpenSSLEngineFactory());
+//```
 
-The following sections describe how you can configure the Java client for
-blue-green and disaster recovery scenarios.
+=== Configure Hazelcast {hazelcast-cloud}
 
-=== Configuring Using CNAME
-
-Using CNAME, you can change the hostname resolutions and use them dynamically.
-Let's describe the configuration with examples.
+// check standalone
```

Comment:
I don't think cloud is implemented

Reply:
Thanks for confirming - standard disclaimer added